lib/grandpa/message_tracker.go

```diff
@@ -19,25 +19,26 @@ type tracker struct {
 	blockState BlockState
 	handler    *MessageHandler
 	votes      votesTracker
-
-	// map of commit block hash to commit message
-	commitMessages map[common.Hash]*CommitMessage
-	mapLock        sync.Mutex
-	in             chan *types.Block // receive imported block from BlockState
-	stopped        chan struct{}
+	commits    commitsTracker
+	mapLock    sync.Mutex
```

Comment:
`mapLock` refers to `catchUpResponseMessages` map? if so I think we can change to `catchUpMessagesLock`

Reply:
Out of scope, changes show up since formatting (alignment) changed.

But... it looks like `mapLock` is for all the 'sub-trackers'/sub-maps. The commit tracker is not thread safe so I would guess we need this `mapLock` for it as well.